### ISSUE TITLE
"org.freedesktop.Platform.GL.default/aarch64/19.08 not installed", followed by the app crashing

### ISSUE THREAD

**Issue by nehemiagurl** [flathub/com.tutanota.Tutanota#199]
Got the following error when installing the app from Discover:
> org.freedesktop.Platform.GL.default/aarch64/19.08 not installed

However, the app is listed as if it is installed:
![Screenshot_20240821_174643](https://github.com/user-attachments/assets/ad6cbcf8-31d0-42dd-9cd9-7f4805767b8e)
Trying to open it results either in instant crash, or in the app crashing then immediately reopening in an infinite loop that can only be stopped by killing the process.
Here's the screen recording I have, sadly it seems to have been somewhat broken by the infinitely spawning process.
[Screencast_20240821_175118.webm](https://github.com/user-attachments/assets/a2b63f8e-7aa9-4125-bfe1-0421b6ede15e)

Running
`flatpak install org.freedesktop.Platform.GL.default/aarch64/19.08`
results in
>Skipping: org.freedesktop.Platform.GL.default/aarch64/19.08 is already installed

Removing the flatpak, installing `org.freedesktop.Platform/aarch64/19.08` then reinstalling the flatpak results in a different error:
> Can't remove org.freedesktop.Platform/aarch64/19.08, it is needed for: com.tutanota.Tutanota
Skipping org.freedesktop.Platform.Locale/aarch64/19.08 due to previous error
and running the app (which again appears installed) results in the exact same behaviour.

### system information:
Operating System: Fedora Linux Asahi Remix 40
KDE Plasma Version: 6.1.4
KDE Frameworks Version: 6.5.0
Qt Version: 6.7.2
Kernel Version: 6.10.5-400.asahi.fc40.aarch64+16k (64-bit)
Graphics Platform: Wayland

**Comment by charlag** [comment]
There is no build for aarch64, if you see it it must be an ancient version

**Comment by nehemiagurl** [comment]
I see. is there a way to remove the aarch64 build so it doesn't appear as available?

**Comment by charlag** [comment]
no, we can't do that. would be great to add aarch64 support though
